src/frequenz/sdk/timeseries/logical_meter/_logical_meter.py

```diff
@@ -33,32 +33,23 @@ class LogicalMeter:
         from datetime import timedelta
 
         from frequenz.sdk import microgrid
-        from frequenz.sdk.timeseries import ResamplerConfig
+        from frequenz.sdk.actor import ResamplerConfig
+        from frequenz.client.microgrid import ComponentMetricId
+
 
         await microgrid.initialize(
-            "grpc://127.0.0.1:50051",
-            ResamplerConfig(resampling_period=timedelta(seconds=1))
+            "microgrid.sandbox.api.frequenz.io:62060",
```

Comment:
It does needs the `grpc` schema. :grimacing:

Really not trying to sound like a :trollface:, but since you are fixing the example, maybe it would be a good idea to test the example to make sure also the sandbox IP and port are correct.

```suggestion
            "grpc://microgrid.sandbox.api.frequenz.io:62060",
```

Not sure if it was the best idea, I was in doubt if using `http(s)` if grpc normally use HTTP as transport, but I thought this way it was going to be more obvious that it is gRPC explicitly.